Office 365/o365/ingest/parser.yml

```diff
@@ -254,6 +256,7 @@ stages:
           office365.device.is_compliant: '{% for deviceProperty in json_event.message.DeviceProperties %}{% if deviceProperty.Name == "IsCompliant" %}{% if deviceProperty.get("Value") != None and deviceProperty.Value != "<Null>"%}{{deviceProperty.Value}}{% endif %}{% endif %}{% endfor %}'
           office365.device.is_compliant_and_managed: '{% for deviceProperty in json_event.message.DeviceProperties %}{% if deviceProperty.Name == "IsCompliantAndManaged" %}{% if deviceProperty.get("Value") != None and deviceProperty.Value != "<Null>"%}{{deviceProperty.Value}}{% endif %}{% endif %}{% endfor %}'
           office365.device.trust_type: '{% for deviceProperty in json_event.message.DeviceProperties %}{% if deviceProperty.Name == "TrustType" %}{% if deviceProperty.get("Value") != None and deviceProperty.Value != "<Null>"%}{{deviceProperty.Value}}{% endif %}{% endif %}{% endfor %}'
+          office365.device.browser_type: '{% for deviceProperty in json_event.message.DeviceProperties %}{% if deviceProperty.Name == "BrowserType" %}{% if deviceProperty.get("Value") != None and deviceProperty.Value != "<Null>"%}{{deviceProperty.Value}}{% endif %}{% endif %}{% endfor %}'
```

Comment:
According to the test, [`user_agent.name`](https://www.elastic.co/guide/en/ecs/8.11/ecs-user_agent.html#field-user-agent-name) seems to be more suitable.

Reply:
Yes, using it didn't show the value. because of `user_agent.original` field